src/data/services/StudioApiService.js

```diff
@@ -8,11 +8,25 @@ ensureConfig([
 const studioBaseUrl = getConfig().STUDIO_BASE_URL;
 
 class StudioApiService {
+  static getServiceUrl(courseID) {
```

Comment:
I think this name is a misnomer; this url isn't really the service url, per se. As we start to build out more APIs on the Studio side, we'll have to change this name to accommodate other endpoints.

It might make sense to just construct the urls as we need them in `getProctoredExamSettingsData` and `saveProctoredExamSettingsData` (perhaps not as DRY, but the scope is relatively limited, so I feel that it's okay) or rename this to be more specific to proctored exam settings (e.g. `getProctoredExamSettingsUrl`).

Reply:
I took the suggestion with function name rename at `getProctoredExamSettingsUrl`